tailwind.config.ts

```diff
@@ -1,5 +1,6 @@
 import type { Config } from "tailwindcss";
 import { fontFamily } from "tailwindcss/defaultTheme";
+import backdropFilter from 'tailwindcss-backdrop-filter';
```

Comment:
This is the method for Tailwind CSS v2, which is now long outdated. Please run the dev server to catch issues like this.